### PR TITLE
Document and purge LoaderConfigConsoleMode and LoaderEntryLastBooted

### PR DESCRIPTION
**Pull request by evelikov**
Hey everyone,

As mentioned in my ASG 2023 talk "A story of a bootloader^W^Wthree bootloaders" some variables are not documented. So here are a few patches fixing that, while also ensuring they are purged on `bootctl remove".

### CODE REVIEW

File: man/systemd-boot.xml [hunk 1]
```diff
@@ -461,6 +461,23 @@
...
+        <listitem><para>The identifier of the boot loader entry last attempted. Set and read by the boot loader.
+        The boot loader modifies <varname>LoaderEntryLastBooted</varname> on every boot, unless
+        <varname>LoaderEntryOneShot</varname> is set.</para>
+
+        <para>The boot loader reads the variable, which takes higher priority than
+        <varname>LoaderEntryDefault</varname>. The variable is ignored when <varname>LoaderEntryOneShot</varname>
+        is set.</para>
+
+        <para><varname>LoaderEntryLastBooted</varname> cannot be used as indication that the last boot was
+        succesful or not.</para>
```
Comment (bluca) [inline man/systemd-boot.xml]: "successful"

**Review by bluca — APPROVED** [review]

File: man/systemd-boot.xml (hunk 1 above)
Comment (evelikov) [reply man/systemd-boot.xml]: Fixed, thanks.

File: man/systemd-boot.xml [hunk 2]
```diff
@@ -461,6 +461,23 @@
...
+      <varlistentry>
+        <term><varname>LoaderEntryLastBooted</varname></term>
+
+        <listitem><para>The identifier of the boot loader entry last attempted. Set and read by the boot loader.
+        The boot loader modifies <varname>LoaderEntryLastBooted</varname> on every boot, unless
+        <varname>LoaderEntryOneShot</varname> is set.</para>
+
+        <para>The boot loader reads the variable, which takes higher priority than
+        <varname>LoaderEntryDefault</varname>. The variable is ignored when <varname>LoaderEntryOneShot</varname>
+        is set.</para>
```
Comment (poettering) [inline man/systemd-boot.xml:477]: note this is only used if "default" is set to "@saved" in loader.conf. This deserves notice.

File: man/systemd-boot.xml [hunk 3]
```diff
@@ -452,6 +461,23 @@
         <xi:include href="version-info.xml" xpointer="v240"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LoaderEntryLastBooted</varname></term>
+
+        <listitem><para>The identifier of the boot loader entry last attempted. Set and read by the boot loader.
+        The boot loader modifies <varname>LoaderEntryLastBooted</varname> on every boot, unless
```
Comment (poettering) [inline man/systemd-boot.xml]: this is a bit misleading. we don't update on every boot. we only update it when the user picks a different menu entry. 

this sounds as if we'd flood the nvram with writes needlessly here.

Comment (evelikov) [reply man/systemd-boot.xml]: Added some updates, hope it reads better now.

**Pull request merged**